bundle.yaml

```diff
@@ -1,75 +1,126 @@
-# The machine constraints for each service in this bundle
-# have been commented out so you don't run into quota
-# problems on public clouds. Modify and uncomment the
-# constraints: lines for each service to reflect your
-# deployment before moving to production.
-#
 series: xenial
-description: |
-  Production-grade Canonical Kubernetes cluster featuring logging and monitoring
-tags:
-- kubernetes
-- containers
-services:
-  kubernetes:
-    charm: cs:xenial/kubernetes-8
-    options:
-      version: v1.3.6
-    expose: true
-    num_units: 3
-    annotations:
-      gui-x: '1200'
-      gui-y: '300'
-#    constraints: arch=amd64 cpu-cores=8 mem=8G root-disk=30G
-  etcd:
-    charm: cs:xenial/etcd-8
-    num_units: 3
-    annotations:
-      gui-x: '1200'
-      gui-y: '600'
-#    constraints: arch=amd64 cpu-cores=2 root-disk=12G
-  filebeat:
-    charm: cs:xenial/filebeat-4
-    annotations:
-      gui-x: '900'
-      gui-y: '300'
-  topbeat:
-    charm: cs:xenial/topbeat-4
-    annotations:
-      gui-x: '900'
-      gui-y: '600'
-  kibana:
-    series: trusty
-    charm: cs:trusty/kibana-14
+applications: 
+  filebeat: 
+    charm: "cs:filebeat-4"
+    annotations: 
+      "gui-x": "500"
+      "gui-y": "950"
+  "kubernetes-master": 
+    charm: "cs:~containers/kubernetes-master-0"
+    num_units: 1
+    annotations: 
+      "gui-x": "800"
+      "gui-y": "850"
+    to: 
```

Comment:
We should probably also strip these to: declarations and instead define the constraints at the service level. I'll need to double-check with the docs to ensure that is still proper bundle format.